Source/src/components/ComponentAnimation.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "Component.h"
+#include <resources/ResourceAnimation.h>
```

Comment:
```suggestion
#include "resources/ResourceAnimation.h"
```
We use "" for internal headers and <> for external libraries.